pages/about/groups/ariawg/contribute.md

```diff
@@ -38,18 +38,18 @@ The ARIA WG accepts public comments on publications on the [public-aria](http://
 
 ## Github Issues and pull requests
 
-Technical work is done in a set of GitHub repositories. Work on a given feature may take place in several repositories, and follows the [ARIA Working Group Work Flow](/about/groups/ariawg/workflow/). Repositories maintained by the ARIA Working Group include:
+Technical work is done in a set of GitHub repositories. Pull Requests are open to the main ARIA monorepo. Issues are open to the child spec repos. See below for details. Work follows the [ARIA Process](https://github.com/w3c/aria/blob/main/documentation/process.md) document. Repositories maintained by the ARIA Working Group include:
```

Comment:
@daniel-montalvo On first reading, it was not immediately clear to me which was the "main ARIA monorepo": the repository is named "w3c/aria" below and is not described as "monorepo" nor "main" repository.

Also, do you feel necessary to specify "monorepo" rather than the simpler "repository"?

May I suggest this alternative?

```markdown
Technical work is done in a set of GitHub repositories. Pull Requests are open to the main w3c/aria repository. Issues are open to the child spec repos. See below for details. 

Work follows the [ARIA Process](https://github.com/w3c/aria/blob/main/documentation/process.md) document. 

Repositories maintained by the ARIA Working Group include:
```

Reply:
> @daniel-montalvo On first reading, it was not immediately clear to me which was the "main ARIA monorepo": the repository is named "w3c/aria" below and is not described as "monorepo" nor "main" repository.
> 
> Also, do you feel necessary to specify "monorepo" rather than the simpler "repository"?> 

Not needed.

> May I suggest the following alternative?
> 
> ```
> Technical work is done in a set of GitHub repositories. Pull Requests are open to the main w3c/aria repository. Issues are open to the child spec repos. See below for details. 
> 
> Work follows the [ARIA Process](https://github.com/w3c/aria/blob/main/documentation/process.md) document. 
> 
> Repositories maintained by the ARIA Working Group include:
> ```

Applied.